deployment/ion/ion.tf

```diff
@@ -271,8 +422,51 @@ resource "kubernetes_deployment" "ion-management-api" {
               value = "${var.app_insights_key}"
             },
           ]
+
+          volume_mount {
+            mount_path = "${var.certificate_mount_path}"
+            name       = "ion-management-api"
+            read_only  = true
+          }
+        }
+
+        volume {
+          name = "ion-management-api"
+
+          secret {
+            secret_name = "${kubernetes_secret.ion-management-api.metadata.0.name}"
+          }
         }
       }
     }
   }
 }
+
+output "client_cert" {
+  value     = "${tls_locally_signed_cert.client.cert_pem}"
+  sensitive = true
+}
+
+output "client_key" {
+  value     = "${tls_private_key.client.private_key_pem}"
+  sensitive = true
+}
+
+output "cluster_ca" {
+  value     = "${tls_self_signed_cert.ca.cert_pem}"
+  sensitive = true
+}
+
+output "server_cert" {
+  value     = "${tls_locally_signed_cert.server.cert_pem}"
+  sensitive = true
+}
+
+output "server_key" {
+  value     = "${tls_private_key.server.private_key_pem}"
+  sensitive = true
+}
+
+output "ion_management_endpoint" {
```

Comment:
We should add an output here which is also outputted in the main.tf which is the k8s service's ip address. This will main terraform will wait for the public ip to be assigned before exiting.